pkg/scd/store/cockroach/operational_intents.go

```diff
@@ -199,6 +222,20 @@ func (s *repo) UpsertOperationalIntent(ctx context.Context, operation *scdmodels
 	if err != nil {
 		return nil, stacktrace.Propagate(err, "Failed to convert id to PgUUID")
 	}
+
+	var ovn pgtype.Text // if the OVN is to be generated by the DSS, it must be NULL in DB, not just an empty string
+	if operation.OVN != "" {
+		ovn = pgtype.Text{
+			String: operation.OVN.String(),
+			Valid:  true,
+		}
+	}
```

Comment:
Side question: if we are going to set the `ovn` field only when it was specified by a client, should we possibly name it `client_specified_ovn`?

Alternatively, could we set it in any case and compute the time-based OVN once at insertion time instead of when it is retrieved from the database? Or is it important to be able to distinguish between client-provided and server-generated OVN, and to do this via a null `ovn` field?

Reply:
> Alternatively, could we set it in any case and compute the time-based OVN once at insertion time instead of when it is retrieved from the database? Or is it important to be able to distinguish between client-provided and server-generated OVN, and to do this via a null ovn field?

I did consider that initially, and at the time of making this PR the assumption was that if the provided suffix is not valid, the DSS just ignores it and do as usual.
However that assumption changed so maybe that's to be reconsidered.

Reply:
I've looked into it, but unfortunately there isn't a simple way (i.e. w/o data migration) of computing the OVN at insertion time instead of when it is retrieved. The risk is that with the DSS update the previous OVNs cease to be valid which is of course not acceptable. 
I will update the column however to make it more explicit that this is not the OVN in all cases however.